### ISSUE TITLE
Restore master branch of clouddriver

### ISSUE THREAD

**Issue by gregturn** [spring-cloud/spring-cloud-spinnaker#7]
Depends on #6

When clouddriver is upgraded to latest cf-java-client, stop using forked copy of clouddriver pinned to Jackson 2.6.

**Comment by markpollack** [comment]
No longer relevant as we are going to use our own uber-jar artifacts via #2
